functions-methods.js

```diff
@@ -21,16 +26,49 @@
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
 
+function typeOfEmail(email2){
+    let domain2 = email2.split('@');
+
+    if(domain2[1] === 'novi-education.nl'){
+        domain2 = 'Student';
+    } else if (domain2[1] === 'novi.nl'){
+        domain2 = 'Medewerker';
+    } else {
+        domain2 = 'Extern';
+    }
+    return domain2;
```

Comment:
Als je niets meer met de waarde gaat doen na de if-else's dan kun je die ook gewoon meteen teruggeven:
```suggestion
    let domain = domain2[1];
    
    if (domain === 'novi-education.nl') {
        return 'Student';
    } else if (domain === 'novi.nl') {
        return 'Medewerker';
    }
    
    return 'Extern';
```